include/cuco/static_map.cuh

```diff
@@ -536,6 +536,47 @@ class static_map {
                                            ValueOut values_out,
                                            cuda_stream_ref stream = {}) const;
 
+  /**
+   * @brief Rebuilds the container.
+   *
+   * @note This function synchronizes the given stream. For asynchronous execution use
+   * `rehash_async`.
+   *
+   * @param stream CUDA stream used for this operation
+   */
+  void rehash(cuda_stream_ref stream = {});
+
+  /**
+   * @brief Rebuilds the container.
+   *
+   * @note This function synchronizes the given stream. For asynchronous execution use
+   * `rehash_async`.
+   * @note Behavior is undefined if the desired `capacity` is insufficient to store all of contained
+   * elements.
+   *
+   * @param capacity New capacity of the container
+   * @param stream CUDA stream used for this operation
+   */
```

Comment:
```suggestion
  /**
   * @brief Reserves at least the specified number of slots and regenerates the map
   *
   * @note Changes the number of slots to a value that is not less than `capacity`, then
   * rehashes the container, i.e. puts the elements into appropriate slots considering
   * that the total number of slots has changed.
   * @note This function synchronizes the given stream. For asynchronous execution use
   * `rehash_async`.
   * @note Behavior is undefined if the desired `capacity` is insufficient to store all of the
   * contained elements.
   *
   * @param capacity New capacity of the container
   * @param stream CUDA stream used for this operation
   */
```

Reply:
Mainly taking from https://en.cppreference.com/w/cpp/container/unordered_set/rehash

The word "rebuild" is ambiguous thus I was expecting a more descriptive explanation.

Reply:
Addressed in [bf38640](https://github.com/NVIDIA/cuCollections/pull/380/commits/bf386409467982168dc497a24ca44afe5c6e977c)